catt/controllers.py

```diff
@@ -528,9 +536,7 @@ class DefaultCastController(CastController, MediaControllerMixin, PlaybackBaseMi
     def __init__(self, cast, name, app_id, prep=None):
```

Comment:
These arguments look suspiciously like we're passing an `app: App` in here, should we just pass that object?

Reply:
Nice catch. Let me fix that.